replay_build_scripts/common.mjs

```diff
@@ -0,0 +1,62 @@
+import { spawnSync } from "child_process";
```

Comment:
Not a leading question, I am genuinely not the familiar with stuff: why do we need the `.mjs` file suffix?

Reply:
This is JS, rather than our other stuff which is all TS. And JS is "very special" when it comes to `import`s. Basically, it marks it as a `Module` rather than a `Script`, which is the "better" of the two options to get `Node` to allow `import`/`export` statements (rather than using `require`).

(Also: Don't get @markerikson started on how utterly frustrating abd annoying this can become, when all you want to do is build a library consumable in all kinds of contexts 😅)

Either way, there is a point to be made to use TS instead of JS here, @klochek

Reply:
There is no ts-node on the build machine :P  Which is fixable!  But is yet-another dependency.  One of the (very very few) reasons  I like shell scripts is that everything is just _there_, on the machine, no need to import anything.  Adding more dependencies, especially build dependencies, is something I strive to avoid.